tests/stdlib/ttypeinfo_std.nim

```diff
@@ -0,0 +1,60 @@
+import typeinfo
+
+type
+  TE = enum
+    blah, blah2
+
+  TestObj = object
+    test, asd: int
+    case test2: TE
+    of blah:
+      help: string
+    else:
+      nil
+
+
+var test = @[0,1,2,3,4]
+var x = toAny(test)
+var y = 78
+x[4] = toAny(y)
+assert x[2].getInt == 2
```

Comment:
you can add `assert x[2].getInt == 2` but why remove
`assert cast[ptr int](x[2].value)[] == 2` ?

Reply:
Because `Any.value` is not exported and `getInt` is the alternative way to get the value.